lib/rules/no-html-safe.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const ERROR_MESSAGE = 'Do not use htmlSafe.';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'disallow the use of htmlSafe',
+      category: 'Miscellaneous',
+      recommended: false,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-html-safe.md',
+    },
+    fixable: null,
+    schema: [],
+  },
+
+  ERROR_MESSAGE,
+
+  create(context) {
+    return {
+      ImportDeclaration(node) {
+        if (
+          (node.source.value === '@ember/string' || node.source.value === '@ember/template') &&
+          node.specifiers.some((s) => s.local.name === 'htmlSafe')
```

Comment:
This should actually be checking `imported` not `local`.

And an you add this invalid test case to ensure this is caught still:

```js
import { htmlSafe as myCustomNameForHtmlSafe } from '@ember/template';
```